Smashing branch

Alaska01 opened the pull request:
-Made thought on visual hierarchy, typography and design principles for this project.
- Created a grayscale heat map of the website that indicates which elements have the greatest weight in the visual hierarchy.
-Created a new HTML document.
-Made background colors of those divs different shades of gray which correspond to their visual weight on the Smashing homepage.
-Copy the basic structure of the page using empty semantic HTML elements that are appropriately sized and positioned.

Alaska01 committed:
initialized the navbar

Alaska01 committed:
created the template areas

Alaska01 committed:
corrected the wrong template styling format

Alaska01 committed:
Styling the grid-one section

Godswilly committed:
Made changes to the stylesheet

Godswilly committed:
Created four grid containers and a flex container

Alaska01 committed:
Formatted the grid section by adding bg colors

Alaska01 committed:
created the main flex section

Alaska01 committed:
created footer and added flex section

Alaska01 committed:
debugging code

Alaska01 committed:
debugging code

Alaska01 committed:
debugging Lint errors

Alaska01 committed:
adding readme file

Alaska01 committed:
worked on arrows

Alaska01 committed:
Made changes to arrows to reflect in githack

Alaska01 committed:
finalized the readme file

andresporras3423 commented:
### **Pull Request Under Review By Oscar Russi (TSE)**

andresporras3423 commented on index.html:
```diff
@@ -1,13 +1,540 @@
+    </div>
+
+    <div class="grid-four dark-gray-bg">
+      <div class="grid-4box-1">
```
Replace div with article tag

andresporras3423 commented on css/styles.css:
```diff
@@ -0,0 +1,515 @@
+.col-2a {
+  grid-column: 2/-1;
+  grid-row: 1/-1;
+  background-color: #333; /* background-color: rgba(3, 3, 3, 0.3); */
```
Remove commented code in this line

andresporras3423 reviewed (changes requested):
### Status: Changes Required :recycle:
**Hello @Alaska01 and @Godswilly** 
Good job with your first submission for this project :muscle: your page meets most of the requirements :clap: :clap:
There are just a few details to improve before you can continue:
- [ ] your README file doesn't have a valid description. Explain whats is this project about.
- [ ] remove commented code in your CSS file
- [ ] Check below, I point in your code where you should replace div with the article tag
- [ ]  For the navbar, use nav element with ul/li inside
- [ ] All the elements and element's backgrounds and font colors should be grayscale color (including images). You can use images in this project, but just grayscale.
- [ ] The footer must contain centered links, check out the original page  :point_down: :point_down: :point_down:: 


![](https://i.imgur.com/cT9rNMkl.png)


Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

Alaska01 committed:
fixed the revision as requested by TSE

Alaska01 committed:
fixed Lint errors

Alaska01 committed:
edited readme

Alaska01 committed:
debugged code

Godswilly committed:
Updated the README file

Alaska01 commented:
Worked on the readme file, removed CSS comments, replaced divs in the navbar with ul/li, centered the footer links, changed the div tag to article tag, and made use of grayscale color for all backgrounds elements and font colors as requested by the TSE.

tirthajyoti-ghosh commented:
# Pull Request under review by T. Ghosh (TSE)

tirthajyoti-ghosh reviewed (changes requested):
# Status: Changes required 🚨
## Impressive project @Alaska01 and @Godswilly  ✨

Great job applying the changes. However, a small part in the `footer` still requires some change.

tirthajyoti-ghosh commented on index.html:
```diff
@@ -1,13 +1,570 @@
+
+  </div>
+   
+  <div class="sub-footer">
```
The footer must have two placeholders for the two images on both sides.
Original 👇
![Annotation 2020-06-06 180247](https://user-images.githubusercontent.com/57726348/83944285-f7056580-a81f-11ea-92c9-26c36930b953.jpg)

Right now 👇
![Annotation 2020-06-06 180209](https://user-images.githubusercontent.com/57726348/83944294-02589100-a820-11ea-85d5-380a08f5b730.jpg)

Alaska01 replied:
Thanks for the review. Have updated the two images in the footer section. Please note that the smashing magazine images are changing from time to time as the site is opened. We have updated two images to demonstrate that the images can be positioned in that position.

tirthajyoti-ghosh reviewed (commented):
Apply this small change and request for another review.

Happy coding 👨‍💻

Alaska01 committed:
Added images to footer section

Alaska01 committed:
updated the live link

tirthajyoti-ghosh reviewed (approved):
# Status: Approved ✔

## Congratulations. Your project is approved!! 🙌👏

Charge onto your next project 🚀

Good luck!! 👍
